tests/plugins/renderer.test.ts

```diff
@@ -297,12 +362,12 @@ describe("renderer", () => {
       sheets: [
         {
           id: 1,
-          cols: { 0: { size: 5 }, 2: { size: 25 } },
+          cols: { 0: { size: 6 }, 2: { size: 26 } },
           colNumber: 3,
           cells: {
             A1: { content: "123456" },
             A2: { content: "=A1" },
-            C1: { content: "123456" },
+            C1: { content: "123458" },
```

Comment:
Why did you changed these values ? (same for the setCellContent below)

Reply:
To have an easier time at debugging and not have 2 cells with the same value.